src/partials/teachers-section.html

```diff
@@ -0,0 +1,68 @@
+<section class="teachers" id="teachers">
+    <div class="container">
+        <h2 class="section-title">Meet our teachers</h2>
+        <p class="section-description">Discover our team of dedicated educators, committed to guiding you on your journey to English proficiency with personalized and engaging instruction</p>
+        <ul class="teachers-list">
+            <li class="teachers-item">
+                <div class="teachers-card">
+                <img 
+                class="teachers-img"
+                srcset="/src/images/teachers/maria.webp 1x, 
```

Comment:
Add `.` before each path, e.g. `./src/images/teachers/maria.webp 1x` and so on. Do it for all paths in the document

Reply:
done